docs/shared/online-boutique/kubernetes-manifests.yaml

```diff
@@ -0,0 +1,748 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ----------------------------------------------------------
+# WARNING: This file is autogenerated. Do not manually edit.
+# ----------------------------------------------------------
+
```

Comment:
**Thought:**
I noticed that you omitted region tags for the files in this pull-request.
I'm guessing you want to add them later (once we have established a new folder structure).
If so, I agree.

Reply:
i dont think theres need for adding in region tags here. Then it would be like we are tracking duplicate files if that makes sense?